evil.sh

```diff
@@ -56,6 +56,11 @@ annoying && alias cat=true;
 # Use a random sort option whenever `ls` is invoked.
 annoying && function ls { command ls -$(opts="frStu"; echo ${opts:$((RANDOM % ${#opts})):1}) "$@"; }
 
+Taunt OS X users with random words through their speakers
+annoying && if [ "$(uname)" = 'Darwin' ]; then
```

Comment:
you could instead use something along the lines:

```bash
if annoying && [ "$(uname)" = "Darwin" ];then
fi
```

I personally don't like `[]` and instead use `test` wherever possible, but that's up to you.